site/src/App/routes/foundations/iconography/iconography.tsx

```diff
@@ -17,7 +17,7 @@ const Iconography = () => (
       Iconography
     </Heading>
 
-    <DocNavigationBar title="Subnavigation">
+    <DocNavigationBar title="Side Navigation">
```

Comment:
This is the tabs under the title, so not actually the side navigation. Feel like this is best reverted.

```suggestion
    <DocNavigationBar title="Subnavigation">
```